modules/getting-started-cli-deploying-first-image.adoc

```diff
@@ -1,21 +1,23 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-cli.adoc
+// * tutorials/dev-app-cli.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-cli-deploying-first-image_{context}"]
-= Deploying your first image
+= Deploying the front-end application
 
-The simplest way to deploy an application in {product-title} is to run an existing container image. The following procedure deploys a front-end component of an application called `national-parks-app`. The web application displays an interactive map. The map displays the location of major national parks across the world.
+The simplest way to deploy an application in {product-title} is to run a provided container image.
+
+The following procedure deploys `parksmap`, which is the front-end component of the `national-parks-app` application. The web application displays an interactive map of the locations of national parks across the world.
 
 .Prerequisites
 
-* You must have access to an {product-title} cluster.
-* Install the OpenShift CLI (`oc`).
+* You have access to an {product-title} cluster.
+* You have installed the OpenShift CLI (`oc`).
 
 .Procedure
 
-* To deploy an application, enter the following command:
+* Deploy the `parksmap` application by running the following command:
 +
 [source,terminal]
 ----
```

Comment:
Since this command is fairly long, it might be a better readability experience to break it into multiple lines: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#new

Reply:
I had considered that for one other command to perhaps add callouts, but for consistency I decided to leave them all one liners, if I didn't need callouts